src/Sylius/Bundle/UserBundle/Controller/UserController.php

```diff
@@ -109,7 +109,7 @@ protected function prepareResetPasswordRequest(Request $request, TokenProviderIn
         $formType = $request->attributes->get('_sylius[form]', 'sylius_user_request_password_reset', true);
         $form = $this->createResourceForm($formType, $passwordReset);
 
-        if (in_array($request->getMethod(), array('POST', 'PUT', 'PATCH')) && $form->submit($request, !$request->isMethod('PATCH'))->isValid()) {
+        if (in_array($request->getMethod(), array('POST', 'PUT', 'PATCH')) && $form->handleRequest($request)->isValid()) {
```

Comment:
should be: https://github.com/symfony/symfony/blob/2d48af774522db8e5389a4413f47a291121eca08/src/Symfony/Component/Form/Extension/HttpFoundation/HttpFoundationRequestHandler.php#L116